lib/karen.coffee

```diff
@@ -117,7 +117,14 @@ class MockNode extends MockElement
   getAttribute: (name) ->
     @attributes[name]
 
-  getElementsByTagName: -> []
+  getElementsByTagName: (name) ->
+    found = []
+    for child in @children
+      if child.tagName.toLowerCase() == name.toLowerCase()
+        found.push(child)
+      if child.tagName.toLowerCase() != 'iframe'
```

Comment:
`tagName` is always uppercase, the comparison on row 123 should be against `name.toUpperCase`